### ISSUE TITLE
[Incompatibility] Fabric: JourneyMap crashes the game if Moonrise is enabled

### ISSUE THREAD

**Issue by MVukanichh** [Tuinity/Moonrise#34]
If either an existing or a new world is loaded while having both JourneyMap and Moonrise enabled, the game crashes seconds after it loads, with Journeymap giving an error in the console. 
I've already opened an issue on Journeymap's github issues page, and I've provided logs to them there, so you can use that issue in order to discuss anything with the Journeymap's developers and to access said log files.
Link to the issue is https://github.com/TeamJM/journeymap/issues/802 .

**Comment by jpenilla** [comment]
I wonder if there is some sort of 3-way(+) mod conflict here, since I haven't run into this with Moonrise in ATM10 which has JM
![image](https://github.com/user-attachments/assets/d7e283e7-12bf-4977-9346-1ff2277ba86f)

**Comment by jpenilla** [comment]
Oh, this is on fabric only? That would explain it.

**Comment by MVukanichh** [comment]
> I wonder if there is some sort of 3-way(+) mod conflict here

It is quite possible, since not once have I had more than 2 mods disabled while testing to see which one causes the crashes, and that was when I disabled Terralith and Tectonic, which did not help. Every other time I had just one mod disabled when testing and when I disabled Moonrise it worked immediately, although, this I didn't mention on the Journeymap's issue, it did take quite a while to join the world, some 10ish seconds more than usually, though "The usual" is with the Moonrise enabled, not disabled.
And it's Quilt, yes, sorry for not mentioning it in the title, I actually forgot that both mods are on NeoForge as well.
